Source/WebCore/platform/graphics/gstreamer/mse/MediaPlayerPrivateGStreamerMSE.cpp

```diff
@@ -308,6 +308,9 @@ void MediaPlayerPrivateGStreamerMSE::didPreroll()
 
 const PlatformTimeRanges& MediaPlayerPrivateGStreamerMSE::buffered() const
 {
+    if (!m_source)
+        return PlatformTimeRanges::emptyRanges();
+
     ASSERT_NOT_REACHED();
```

Comment:
I would have done differently:

```
if (m_source)
    ASSERT_NOT_REACHED();

return  PlatformTimeRanges::emptyRanges();
```

And maybe adding a comment, because having this method so useless is weird.

Reply:
That's not allowed by our style guidelines. `ASSERT_NOT_REACHED() statement fallthrough may result in unexpected code execution.  [security/assertion_fallthrough] [4]`

But I can add a comment.